buildmaster/old_new_porting_map.yml

```diff
@@ -8,54 +8,54 @@
 
 # HERA
 HERACOMBCCEM:
-  new_name: HERA_CC_318GEV_EM-SIGMARED
+  new_name: HERA_CC_318GEV_SIGMARED_EM
```

Comment:
Now I'm confused ... just out of old habits I commented at first `SIGMARED_EM` - but then realized my mistake ... shouldn't it be `SIGMARED-EM`?

Reply:
Since they are all `sigmared` for these datasets the choice between `_` and `-` becomes arbitrary.